src/main.rs

```diff
@@ -63,10 +62,11 @@ fn do_main() -> Result<(), Box<Error>> {
         Err(err) => return Err(Box::new(err)),
     };
 
-    let mut statistics = Statistics::new();
+    let statistics = Profiler::new("gir profiler");
```

Comment:
"Timing information for gir profiler:" IMHO no good,
prefer "Timing information for Gir:"